dotcom-rendering/src/web/components/ArticleContainer.tsx

```diff
@@ -76,7 +76,7 @@ const articleAdStyles = css`
 			}
 		}
 	}
-	.ad-slot--fluid {
+	.ad-slot--fluid:not(.ad-slot--im) {
```

Comment:
The `:not(.ad-slot--im)` is included to ensure the inline merch slot doesn't take the full width of the container.

The same functionality is implemented in frontend along with [more rules for other ads](https://github.com/guardian/frontend/blob/07305ca708926ef75db9b4489644c1174585cd84/static/src/stylesheets/module/_adslot.scss#L443-L445). I've neglected to copy it verbatim as there seems little value in doing that without considering whether we really want those rules here.